lib/inspec/objects/input.rb

```diff
@@ -3,19 +3,8 @@
 require 'utils/deprecation'
 
 module Inspec
+  # TODO: move this
```

Comment:
Could you link an issue or add some more context?

Reply:
It will be moved on the next PR, when #3761 is implemented.